devsecops/axe-testing/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "axe-testing",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "lint": "eslint src",
+    "dbg": "node --experimental-vm-modules --inspect-brk node_modules/.bin/jest --runInBand --no-cache",
+    "prestart:docker": "mkdir -p axe-results && sudo chmod 777 axe-results && docker build -t axe .",
+    "start": "node --experimental-vm-modules index.js",
+    "start:docker": "docker run --rm --network host -v ${PWD}/axe-results:/app/axe-results -e COMMIT_SHA=$(git rev-parse HEAD) axe",
+    "test:unit": "NODE_OPTIONS=--experimental-vm-modules jest ./src/*.test.js --coverage",
+    "test:e2e:startServer": "npx http-server ./tests/test-pages -p 8080 & echo $! > server.pid",
+    "test:e2e:waitForSeverToStart": "for i in {1..10}; do nc -z localhost 8080 && break || echo 'Waiting for server to be ready...'; sleep 1; done",
+    "test:e2e:runTests": "NODE_OPTIONS=--experimental-vm-modules npx jest tests/e2e.test.js",
+    "test:e2e:stop": "kill $(cat server.pid) || true",
+    "test:e2e": "npm run test:e2e:startServer && npm run test:e2e:waitForSeverToStart && npm run test:e2e:runTests && npm run test:e2e:stop",
```

Comment:
Should be able to start the http server as a pre-test step inside the e2e test, and clean it up at the end, I think!